Hunting Queries/BehaviorAnalytics/Anomalous Sign-in Activity.yaml

```diff
@@ -20,9 +19,7 @@ query: |
 
  BehaviorAnalytics
  | where ActionType == "Sign-in"
- | where UsersInsights.IsNewAccount == True or UsersInsights.IsDormantAccount == True
- or ActivityInsights.FirstTimeUserAccessedResource == True and ActivityInsights.ResourceUncommonlyAccessedAmongPeers == True
- or ActivityInsights.FirstTimeUserUsedApp == True and ActivityInsights.AppUncommonlyUsedAmongPeers == True
+ | where UsersInsights.IsNewAccount == True or UsersInsights.IsDormantAccount == True or ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done